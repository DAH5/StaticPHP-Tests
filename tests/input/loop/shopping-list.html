---
title: Shopping List
description: NOTE TO SELF: What to get when grocery shopping.
---
<ul>
    <li>Bread</li>
    <li>Butter</li>
    <li>Milk</li>
    <li>Water</li>
</ul>