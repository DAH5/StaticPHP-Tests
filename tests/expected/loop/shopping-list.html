<ul>
    <li>Bread</li>
    <li>Butter</li>
    <li>Milk</li>
    <li>Water</li>
</ul>